L1Trigger/Phase2L1ParticleFlow/src/newfirmware/puppi/linpuppi_ref.h

```diff
@@ -138,6 +143,11 @@ namespace l1ct {
                           const PVObjEmu &pv,
                           const std::vector<PFChargedObjEmu> &pfch /*[nTrack]*/,
                           std::vector<PuppiObjEmu> &outallch /*[nTrack]*/) const;
+    //vtx vetor
+    void linpuppi_chs_ref(const PFRegionEmu &region,
+                          const std::vector<PVObjEmu> &pv /*[nVtx]*/,
+                          const std::vector<PFChargedObjEmu> &pfch /*[nTrack]*/,
+                          std::vector<PuppiObjEmu> &outallch /*[nTrack]*/) const;
```

Comment:
you forgot to update also this file changing the function instead of adding the new one

Reply:
fixed